packages/playwright/src/matchers/matcherHint.ts

```diff
@@ -20,6 +20,14 @@ import type { Locator } from 'playwright-core';
 import type { StackFrame } from '@protocol/channels';
 import { stringifyStackFrames } from 'playwright-core/lib/utils';
 
+const kNoElementsFoundMessage = '<element(s) not found>';
+
+export function rewriteIfNoElementFound(received: any, log: string[] | undefined): any {
+  if (log?.[log.length - 1].startsWith('waiting for '))
```

Comment:
I thought you'd still set `<element(s) not found>` on the server, and check the "received" here to avoid rewriting it into something on the client-side. This way, all language ports will also get it for free.